.readthedocs.yml

```diff
@@ -1,6 +1,41 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
+# for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: dirhtml
+  configuration: docs/conf.py
+  fail_on_warning: true  # https://github.com/pypa/setuptools/issues/2424
+
+# Build documentation with MkDocs
```

Comment:
Let's omit commented template code. The reference to the docs at the top should be sufficient.